### PR TITLE
Update pull request template

### PR DESCRIPTION
**Pull request by piperlincoln**
Added pull request template, adapted from [here](https://github.com/stevemao/github-issue-templates/edit/master/questions-answers/PULL_REQUEST_TEMPLATE.md) and [here](https://github.com/stevemao/github-issue-templates/edit/master/conversational/PULL_REQUEST_TEMPLATE.md). (Specified in #22).

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+If it fixes an open issue, please link to the issue here. 
+
+## Changes
+What kind of change does this PR introduce? (Bug fix, feature, documentation update...)
+
+## Behavior
+What is the current behavior? What is the new behavior?
+
+## Checklist
+- [ ] My code follows the code style of this project.
```
Comment (kkiesling) [inline PULL_REQUEST_TEMPLATE.md]: I'm not sure all the things in this checklist are applicable to this specific project.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Description
```
Comment (kkiesling) [inline PULL_REQUEST_TEMPLATE.md:4]: I'm not sure we have to be as prescribed for this PR template. Perhaps a note at the top "Please be sure your PR addresses the following:" can help make it more open, but still provide good direction.

**Review by kkiesling — COMMENTED** [review]
Other PR templates I've seen have been a little less prescribed than this. Do we know if this is the text that shows up in the PR message box when someone creates a new PR? I've been on projects where if you make a PR, the text box has some guidelines in it with a note that says "delete this text and add your own description."

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (piperlincoln) [reply PULL_REQUEST_TEMPLATE.md]: You're right! I removed the checklist as I felt it was unnecessary.

**Comment by piperlincoln** [comment]
@kkiesling I looked into it, and this text does appear in the PR message box when a new PR is created! I have removed the checklist portion, as I felt it was just repeating information already covered. I also included a note at the top to the user to make it more open but still provide direction.

**Review by kkiesling — APPROVED** [review]
I'll wait a few days for others to comment before merging.

**Pull request merged**